From: Bart Van Assche <bvanassche@acm.org>
Date: Fri, 18 Nov 2011 18:40:52 +0100
Subject: [PATCH 16/77] hv: Add Kconfig menu entry

commit 124a6b4c4dae0317a0689a42ed0c68fabeb10c8d upstream.

Most of the drivers/*/Kconfig files define a menu entry. Define
a menu item for hv too such that it becomes uniform with e.g.
virtio for at least "make xconfig" and "make menuconfig" users.

Signed-off-by: Bart Van Assche <bvanassche@acm.org>
Cc: Greg Kroah-Hartman <gregkh@suse.de>
Cc: James Bottomley <JBottomley@Parallels.com>
Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/hv/Kconfig |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: linux-3.2.46/drivers/hv/Kconfig
===================================================================
--- linux-3.2.46.orig/drivers/hv/Kconfig	2013-07-26 18:56:15.000000000 +0000
+++ linux-3.2.46/drivers/hv/Kconfig	2013-07-26 19:18:15.000000000 +0000
@@ -1,3 +1,5 @@
+menu "Microsoft Hyper-V guest support"
+
 config HYPERV
 	tristate "Microsoft Hyper-V client drivers"
 	depends on X86 && ACPI && PCI
@@ -11,4 +13,4 @@
 	help
 	  Select this option to enable the Hyper-V Utilities.
 
-
+endmenu
